source/server/util.js

```diff
@@ -69,52 +71,106 @@ export function getCurrentRoundState(gameInstance) {
   ];
 } /* getCurrentRoundState */
 
+function getRandomSuite(){
+  let suites = [SUITES.WANDS, SUITES.CUPS, SUITES.SWORDS, SUITES.PENTACLES];
+
+  return suites[Math.floor(Math.random() * suites.length)];
+}
+
 /**
  * Determines the multiplier for a given matchup between two cards
  * (Positive multiplier implies that card1 has a suite-based advantage over card2)
  * @param { Types.Card } card1 card to compare, target of multiplier
  * @param { Types.Card } card2 card to compare
+ * @param {String} worldEvent current worldEvent for the round
  * @returns { Types.Card } matchup multiplier for card1 with reference to card2
  */
-function getMultiplier(card1, card2) {
-  switch (card1.suite) {
+function getMultiplierWorldEvent(card1, card2, worldEvent) {
+
+  let multiplier = MULTIPLER.NEUTRAL;
+  let suite1 = worldEvent === WORLD_EVENTS.RANDOM_SUITE ? getRandomSuite() : card1.suite;
+  let suite2 = worldEvent === WORLD_EVENTS.RANDOM_SUITE ? getRandomSuite() : card2.suite;
+
+  switch (suite1) {
     case SUITES.WANDS:
-      if (card2.suite === SUITES.CUPS) return 2;
-      else if (card2.suite == SUITES.SWORDS) return 0.5;
-      return 1;
+      if (suite2 === SUITES.CUPS) multiplier = worldEvent === WORLD_EVENTS.SUITE_BOOST_WANDS ? MULTIPLER.GREATER + MULTIPLER.BOOST : MULTIPLER.GREATER;
+      else if (suite2 == SUITES.SWORDS) multiplier = worldEvent === WORLD_EVENTS.SUITE_BOOST_WANDS ? MULTIPLER.LESS + MULTIPLER.BOOST : MULTIPLER.LESS;
     case SUITES.CUPS:
-      if (card2.suite === SUITES.SWORDS) return 2;
-      else if (card2.suite == SUITES.WANDS) return 0.5;
-      return 1;
+      if (suite2 === SUITES.SWORDS) multiplier =  worldEvent === WORLD_EVENTS.SUITE_BOOST_CUPS ? MULTIPLER.GREATER + MULTIPLER.BOOST : MULTIPLER.GREATER;
+      else if (suite2 == SUITES.WANDS) multiplier =  worldEvent === WORLD_EVENTS.SUITE_BOOST_CUPS ? MULTIPLER.LESS + MULTIPLER.BOOST : MULTIPLER.LESS;
     case SUITES.SWORDS:
-      if (card2.suite === SUITES.WANDS) return 2;
-      else if (card2.suite == SUITES.CUPS) return 0.5;
-      return 1;
+      if (suite2 === SUITES.WANDS) multiplier =  worldEvent === WORLD_EVENTS.SUITE_BOOST_SWORDS ? MULTIPLER.GREATER + MULTIPLER.BOOST : MULTIPLER.GREATER;
+      else if (suite2 == SUITES.CUPS) multiplier =  worldEvent === WORLD_EVENTS.SUITE_BOOST_SWORDS ? MULTIPLER.LESS + MULTIPLER.BOOST : MULTIPLER.LESS;
+    case SUITES.PENTACLES:
+      multiplier = worldEvent === WORLD_EVENTS.SUITE_BOOST_PENTACLES ? MULTIPLER.NEUTRAL + MULTIPLER.BOOST : MULTIPLER.NEUTRAL;
     default:
-      return 1;
+
   }
+
+  return worldEvent === WORLD_EVENTS.SUITE_REVERSED ? 1/multiplier : multiplier;
+
 } /* getMultiplier */
 
+function getRandomValue() {
```

Comment:
Might not need a JSDOC comment because of its minimal functionality, but consistency in documentation could help.